labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -40,7 +40,7 @@ object CollectionExercise01 {
 
 object CollectionExercise02 {
 
-  class Person(val age: Int, val name: String)
+  case class Person(val age: Int, val name: String)
```

Comment:
At this point in the exercise we haven't covered case classes yet, so we'd prefer to postpone that to later.